UI: Finish ElasticSearch search/sort/page for Vertices

kfoss opened the pull request:
Rewrite VertexListCtrl to support main ES operations
- TODO: re-incorporate edge following
- TODO: mirror to EdgeListCtrl

Commit:
UI: Finish ElasticSearch search/sort/page for Vertices

Rewrite VertexListCtrl to support main ES operations
- TODO: re-incorporate edge following
- TODO: mirror to EdgeListCtrl

erickt commented:
Looks good to me. I'll leave it up to charlie if he wants to merge this in.

cglewis commented:
I noticed that Name and Age are hard coded.  This is not going to work so well for graphs without those fields.  How much effort would it be to switch that to say the results of _mapping from elasticsearch?

Commit:
UI: Build table columns based on ElasticSearch results

Update VertexListCtrl to dyamically build table columns
based on ElasticSearch sort results.

kfoss commented:
Updated controller to dynamically build columns based on ElasticSearch results.  Moving off the AngularJS sorting function also speeds up page load.

Commit:
UI: Fix table header template to support groupBy method

cglewis commented:
LGTM.
